src/typeDefs/typeDefs.ts

```diff
@@ -0,0 +1,27 @@
+import { gql } from 'apollo-server-core';
+import { constructionSiteGQL } from './constructionSite';
+import { dateGQL } from './date';
+import { locationGQL } from './location';
+import { bodyGQL } from './OParl';
+import { roadworkGQL } from './roadwork';
+
+const _baseGQL = gql`
+  type Query {
+    roadworks: [Roadwork!]!
+  }
+
+  type Mutation {
+    createRoadwork(name: String!, description: String): Roadwork!
+    deleteRoadwork(id: ID!): Roadwork!
+    updateRoadwork(id: ID!, name: String, description: String): Roadwork!
+  }
+`;
+
+export const typeDefs = [
+  _baseGQL, // query and mutation need to be first, because their types are extended by others
```

Comment:
doesn't the underscore indicate something unused, which isn't the case here?

Reply:
i have come across multiple conventions using the underscore prefix for various reasons.

i only know it to mean "unused variable" if the name consists of exclusively underscores.

in this case i just wanted to make this one variable stand out among the rest and be correctly sorted to the beginning as well.

if you have a better suggestion feel free to leave it here.

Reply:
good to know, well reasoned